src/dmd/dsymbolsem.d

```diff
@@ -4751,6 +4751,9 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
         if (sd.inv)
             reinforceInvariant(sd, sc2);
 
+        // recalculate isPOD
+        sd.ispod = StructPOD.fwd;
+
```

Comment:
This is a bit disturbing, as it suggests that isPOD can return different values for the same struct depending on where in the semantic analysis it is.

The most reliable way to deal with this is to have isPOD fail if semantic analysis has not proceeded far enough to get an accurate answer. Next best would be to have isPOD fail if it produces different results as semantic analysis proceeds. Worst would be to silently produce different results.

Reply:
It does actually return different results in the test case I provided.